### ISSUE TITLE
Documentation enhancement

### ISSUE THREAD

**Issue by dfguerrerom** [12rambau/sepal_ui#489]
I would like to improve the current documentation structure, especially in the following elements:

- [ ] - Alphabetically sort example and content categories
- [ ] - Categorize the categories in the cookbook section
- [ ] - Look for broken links 404 and fix them
- [ ] - Add drawer interaction (state icons) examples and usage in-app section

@12rambau, in the content section, the main classes are not showing the mandatory parameters, only a rough reference to `**kwargs`, does it respond to a reason?

what do you think?

**Comment by 12rambau** [comment]
Yes to everything (alphabetical order is just a rgument to add to toctree directives)

For the kwargs I don't really know and it's also bugging me 😄
